pkg/platform/runtime/setup/setup.go

```diff
@@ -110,6 +111,8 @@ type Targeter interface {
 
 	// OnlyUseCache communicates that this target should only use cached runtime information (ie. don't check for updates)
 	OnlyUseCache() bool
+	// OnlyInstallFromDir communicates that this target should only install artifacts from the given directory (i.e. offline installer)
+	OnlyInstallFromDir() *string
```

Comment:
Can we name this `CacheDir()` instead? That's effectively what it is. The `OnlyInstallFrom` naming just throws my brain for a loop.